system/include/emscripten/fetch.h

```diff
@@ -88,7 +88,7 @@ typedef struct emscripten_fetch_attr_t {
 
   // Indicates whether cross-site access control requests should be made using
   // credentials.
-  EM_BOOL withCredentials;
+  bool withCredentials;
```

Comment:
Should we include `em_types.h` in this file?

Reply:
Its already transitively included via `emscripten/html5.h` (as before).

Reply:
Updated, replacing `emscripten/html5.h` with `em_types.h`.. cleaner!